pkg/apiserver/apiserver.go

```diff
@@ -36,9 +41,12 @@ func (s ServiceCatalogAPIServer) PrepareRun() RunnableServer {
 // DefaultAPIResourceConfigSource returns a default API Resource config source
 func DefaultAPIResourceConfigSource() *serverstorage.ResourceConfig {
 	ret := serverstorage.NewResourceConfig()
-	ret.EnableVersions(
-		servicecatalogv1beta1.SchemeGroupVersion,
-	)
+	versions := []schema.GroupVersion{servicecatalogv1beta1.SchemeGroupVersion}
+
+	// if utilfeature.DefaultFeatureGate.Enabled(features.PodPreset) {
+	versions = append(versions, settingsv1alpha1.SchemeGroupVersion)
+	// }
```

Comment:
excess feature gate done somewhere else?

Reply:
oops, I had commented that out :) Will take care of that.